app/views/dashboard/_discovery_widget_host_list.html.erb

```diff
@@ -3,7 +3,7 @@
   <th><%= _('Host') %></th>
   <th><%= _('Model') %></th>
   <th><%= _('IP Address') %></th>
-  <th><%= _('CPU') %></th>
+  <th><%= _('CPU count') %></th>
```

Comment:
This will occupy more space and there is not much space there. Mind changing this to `CPUs` both here and in the main discovery list (we have `CPU` there as well).